dashboard/src/components/ChainTimeline.tsx

```
@@ -0,0 +1,363 @@
+  Chip,
+  Divider,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  Badge,
+} from '@mui/material';
+import {
+  ExpandMore,
+  CheckCircle,
+  Error,
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)